server.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const db = require('./data/db');
+const cors = require('cors');
+const port = 5555;
+const server = express();
+server.use(express.json());
+server.use(cors({ origin: 'http://localhost:3000' }));
+server.get('/', (req, res) =>  {
+res.send('hello from express');
+})
+
+server.post('/api/users', (req, res) => {
+if (req.body.name === undefined || req.body.bio === undefined) {
```

Comment:
You could simply this by declaring your const { name, bio } after line 12 and then changing line 13 to `if (!name || !bio)